docs/source/format/CDataInterface.rst

```diff
@@ -218,6 +218,16 @@ names and types of child fields are read from the child arrays.
 | ``+us:I,J,...``        | sparse union with type ids I,J...                 |            |
 +------------------------+---------------------------------------------------+------------+
 
+Types that are encoded in an alternative layout have format strings ending with ``>``
+followed by a character describing the alternative layout.  For example, an int32 column
+encoded in the stringy-view layout would have the format string ``I>~``.
```

Comment:
This approach could potentially be used in places like RLE as well where data can